ios/PurchasesHybridCommon/PurchasesHybridCommon/ErrorContainer.swift

```diff
@@ -53,4 +61,18 @@ import RevenueCat
 
         self.info = info
     }
+
+    private static func findStoreKitErrorCodeIfAny(_ error: Error) -> NSError? {
+        var currentError: NSError? = error as NSError
+        var storeKitError: NSError?
+        while let underlyingNSError = currentError?.userInfo[NSUnderlyingErrorKey] as? NSError {
+            if !underlyingNSError.domain.starts(with: "RevenueCat") {
```

Comment:
where do we define the `RevenueCat` domain? I couldn't find it in a quick search

Reply:
I believe it uses the module name and the file/class name. In my tests, our domains looked like: `RevenueCat.ErrorCode` or `RevenueCat.ProductsFetcherSK2.Error`

Reply:
I see 👍 thanks! Asking because I remember we used to set it explicitly back in the objc days, guess we moved away from it.